docs/types/value-types.rst

```diff
@@ -499,7 +499,7 @@ terminate the string literal. Newline only terminates the string literal if it i
 Hexadecimal Literals
 --------------------
 
-Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``). Their content must be a hexadecimal string and their value will be the binary representation of those values.
+Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``, ``hex"0011_22_FF"``). Their content must be hexadecimal digits which can optionally be delimited with a single underscore between byte boundaries. The value of the literal will be the binary representation of the hexadecimal number.
```

Comment:
```suggestion
Hexadecimal literals are prefixed with the keyword ``hex`` and are enclosed in double or single-quotes (``hex"001122FF"``, ``hex"0011_22_FF"``). Their content must be hexadecimal digits which can optionally use a single underscore separator between byte boundaries. The value of the literal will be the binary representation of the hexadecimal number.
```

Reply:
And maybe do not use the term `hexadeimal number` because that raises questions of alignment and endianness.

Reply:
Did you undo the change again?

Reply:
Must have accidentally force-pushed it away again, yes... I'll fix it.